content/committees/specification/tckprocess/index.md

```diff
@@ -412,3 +419,11 @@ is not a mechanism to attempt to handle implementation specific issues.
 The only time tests may be added to a TCK are in a major or minor
 release. A service release which updates the exclude list MUST not have
 test additions or changes.
+
+## Process for releasing a point revision of a TCK
+
+The process for releasing a point revision of a TCK entails filing an issue in the [jakartaee/specifications repository with the following details](https://github.com/jakartaee/specifications):
+
+-   Link to the TCK release to be published.
+-   Updated TCK links in the specification's _index.md file.
```

Comment:
We are releasing:
- Jakarta EE Platform TCK 9.0.1
- Jakarta Servlet TCK 5.0.1

I will follow the details mentioned in this pull request in case it gets merged in the future.